lib/network/rest/utils.js

```diff
@@ -0,0 +1,13 @@
+'use strict'; // eslint-disable-line
+
+module.exports.explodePath = function explodePath(path) {
+    const pathMatch = /^(\/[a-zA-Z0-9]+)(\/([0-9a-f]*))?$/.exec(path);
+    if (pathMatch) {
+        return {
+            service: pathMatch[1],
+            key: (pathMatch[3] !== undefined && pathMatch[3].length > 0 ?
+                  pathMatch[3] : undefined),
+        };
+    }
+    return undefined;
```

Comment:
Of course the other side of it is that you *have* to catch at some point to not crash your process, but when using `return` if you're not checking the returned error, you don't control what will be going on later (maybe stuck a client or even take the error contents for payload and return some garbage...)